docs/HelloWorldWalkthrough.md

```diff
@@ -0,0 +1,312 @@
+    <!-- EDL Code Generator properties -->
+    <!-- ********* -->
+    <PropertyGroup Label="EDL Codegen props">
+      <VbsEnclaveEdlPath>$(ProjectDir)..\MySecretEnclave.edl</VbsEnclaveEdlPath>
+      <VbsEnclaveNamespace>VbsEnclave</VbsEnclaveNamespace>
+      <VbsEnclaveVirtualTrustLayer>Enclave</VbsEnclaveVirtualTrustLayer>
+      <VbsEnclaveSkipCodegen>false</VbsEnclaveSkipCodegen>
+    </PropertyGroup>
+        
+    <!-- ********* -->
+    <!-- Property Group for Cert to sign with -->
+    <!-- ********* -->
+    <PropertyGroup Label="Test Signing certificate">
+        <EnclaveCertName>TheDefaultTestEnclaveCertName</EnclaveCertName>
+    </PropertyGroup>
+    
+    <!-- ********* -->
+    <!-- Properties for Post-Build Commands -->
+    <!-- ********* -->
+    <PropertyGroup Label="Post-Build command props">
+        <VEIID_Command>"$(WindowsSDKVersionedBinRoot)\$(PlatformTarget)\veiid.exe" "$(OutDir)$(TargetName)$(TargetExt)"</VEIID_Command> 
+        <SIGNTOOL_Command>signtool sign /ph /fd SHA256 /n "$(EnclaveCertName)" "$(OutDir)$(TargetName)$(TargetExt)"</SIGNTOOL_Command>
+    </PropertyGroup>
+
+1. Add a Post-Build step to the build configurations
+    <!-- ********* -->
+    <!-- Post-Build steps - Apply VEIID protection and sign the enclave dll -->
+    <!-- ********* -->
+    <PostBuildEvent>
+        <Command>$(VEIID_Command)</Command>
+        <Message>Apply VEIID Protection</Message>
+    </PostBuildEvent>
+    <PostBuildEvent>
+        <Command>$(SIGNTOOL_Command)</Command>
+        <Message>Sign the enclave</Message>
+    </PostBuildEvent>
+    ```
+
+1. Save and Reload the project. 
+1. Via Project Properties, set C/C++ compilation settings:
+   * Precompiled headers -> Not using 
+   * Basic runtime checks –> Default
+   * C++ Language Standard -> /std:c++ 20 (due to usage of span)
+   * Conformance mode -> Yes (permissive-)
+   * Library – MultiThreaded Debug /MTd, and /MT for Release builds
+
+1.	Via Project Properties, Set Linker settings:
+    * Enable Incremental Linking -> No (/INCREMENTAL:NO)
+    * Ignore All Default Libraries -> Yes (/NODEFAULTLIB)
+    * Add Additional Dependencies -> $(VBS_Enclave_Dependencies) 
+    * For command line, add “additional options” -> /ENCLAVE /INTEGRITYCHECK /GUARD:MIXED
+1.	Go to solution explorer and open PCH.H and update it:
+      * Remove #include “framework.h”
+      * Add the enclave header, winenclave.h, eg:
+        ```cpp
+        #ifndef PCH_H
+        #define PCH_H
+        
+        // add headers that you want to pre-compile here
+        // #include "framework.h"
+        
+        // DEMO - Step 1: Include the enclave header
+        #include <winenclave.h>
+        
+        #endif //PCH_H
+        ```
+1.	Right click the dll project and choose “Build”. This will generate the projection layer for the enclave dll. You should have a couple initial error messages after code generation due to lack of implementation. We will fix this in the next step.
```

Comment:
>“ [](http://example.com/codeflow?start=42&length=1)

looks like a Word paste - can you change all the double quote things to actual " char?